src/controllers/api.controller.js

```diff
@@ -1,27 +1,31 @@
 import { loadMovies } from "../helpers/movies.js";
 import { loadAllReviews, loadMovieReviews } from "../helpers/reviews.js";
 
-import { getAllScreenings, movieScreenings } from "../helpers/screenings.js";
+import { getAllScreenings, movieScreenings, screeningsStartpage } from "../helpers/screenings.js";
 
 const apiCtrl = {};
 
 apiCtrl.movies = async (req, res) => {
-    res.json(await loadMovies());
-}
+  res.json(await loadMovies());
+};
 
-apiCtrl.movieScreenings = async (req, res) => { 
-    res.json(await movieScreenings(req.params.id));
-}
+apiCtrl.startpageScreenings = async (req, res) => {
```

Comment:
I see no great reason for having these "controllers". I would have just put this code in the route handlers in `api.routes.js`.